src/main.rs

```diff
@@ -63,15 +63,22 @@ impl EventHandler for Handler {
             .guild(guild_id)
             .expect("Could not find guild in cache.");
 
-        let channel_id = guild
-            .voice_states
-            .get(&config.discord_user_id.into())
-            .and_then(|voice_state| voice_state.channel_id);
-        drop(guild);
+        // Check if any of the admins are in the VC
+        let user_list = guild.voice_states;
+
+        let mut channel_ids: Vec<Option<id::ChannelId>> = Vec::new();
```

Comment:
You only need one channel_id later on. Just use an `Option<id::ChannelId>`  and move the is_some() check into the loop in line 71.